http-server-netty/src/test/groovy/io/micronaut/http/server/netty/websocket/BinaryChatClientWebSocket.java

```diff
@@ -71,4 +73,10 @@ public void onMessage(
     public abstract Future<ByteBuf> sendAsync(ByteBuf message);
 
     public abstract Single<ByteBuffer> sendRx(ByteBuffer message);
+
+    public void sendMultiple() {
+        session.sendSync(new TextWebSocketFrame(false, 0, "hello"));
+        session.sendSync(new ContinuationWebSocketFrame(false, 0, " "));
+        session.sendSync(new ContinuationWebSocketFrame(true, 0, "world"));
```

Comment:
One more change is I think a test needs to be added for more than 2 continuation frames since the way the code is written behaves differently depending if there are 2 or more which isn't covered by test I don't think.